supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala

```diff
@@ -13,7 +13,7 @@ object ActionUtils {
   val sshPortMappings = Map("22" -> new Integer(22))
 
   /*
-   * At the moment, the cloud-sdk:slim (727MB on 2019-09-26) and possibly stedolan/jq (182MB) decompressed ~= 1 GB
+   * At the moment, cloud-sdk (924MB for 276.0.0-slim) and stedolan/jq (182MB) decompressed ~= 1.1 GB
```

Comment:
cloud-sdk:slim is pulled and used for localization and delocalization. Although IIRC `gcloud` may now be available on VMs by default? Although we may want to stick with the pinned version of the SDK since we had some bad experiences when we used to float the cloud-sdk version. 

stedolan/jq is when in CWL output processing I believe.

Reply:
Huh, I just checked and looks like Google's own machine image used for PAPI workers really doesn't have cloud sdk on it. I would never expected that 🙂